File: 3131-FindtheIntegerAddedtoArrayI/3131-FindtheIntegerAddedtoArrayI.cpp
// Last updated: 3/10/2026, 11:41:52 AM
1class Solution {
2public:
3    int addedInteger(vector<int>& nums1, vector<int>& nums2) {
4          sort(nums1.begin(),nums1.end());
5          sort(nums2.begin(),nums2.end());
6          return nums2[0] - nums1[0];
7    }
8};

File: 3131-FindtheIntegerAddedtoArrayI/3131-FindtheIntegerAddedtoArrayI.js
// Last updated: 3/10/2026, 11:37:05 AM
1/**
2 * @param {number[]} nums1
3 * @param {number[]} nums2
4 * @return {number}
5 */
6var addedInteger = function(nums1, nums2) {
7    nums1 = nums1.sort((a,b)=>a-b);
8    nums2 = nums2.sort((a,b)=>a-b);
9
10    return nums2[0] - nums1[0]
11};